### PR TITLE
DOCSP-35724: $in operator documentation

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-35724>

### Staging Links

<li><a href=https://deploy-preview-372--mongodb-docs-csharp.netlify.app/fundamentals/atlas-search>fundamentals/atlas-search</a></li>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/atlas-search.txt [hunk 1]
```diff
@@ -335,6 +335,28 @@ The search returns the following document:
...
+
+Use the ``In()`` method to search for documents that have field values matching a list
+of specified values.
+
+The following example searches the ``guitars`` collection for documents that have a
+``make`` field value of either ``"Fender"`` or ``"Gibson"``.
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs
+   :start-after: // start-in-search
+   :end-before: // end-in-search
```
Comment (mayaraman19) [inline source/fundamentals/atlas-search.txt:349]:
```suggestion
   :start-after: start-in-search
   :end-before: end-in-search
```

File: source/fundamentals/atlas-search.txt [hunk 2]
```diff
@@ -335,6 +335,28 @@ The search returns the following document:
 To learn more about the ``geoWithin`` operator, see the :atlas:`geoWithin </atlas-search/geoWithin>` 
 Atlas guide.
 
+In
+~~
+
+Use the ``In()`` method to search for documents that have field values matching a list
+of specified values.
```
Comment (mayaraman19) [inline source/fundamentals/atlas-search.txt:342]:
```suggestion
Use the ``In()`` method to search for documents with field values that match a list
of specified values.
```

**Review by mayaraman19 — APPROVED** [review]
LGTM with a few comments!

File: source/fundamentals/atlas-search.txt (hunk 1 above)
Comment (mcmorisi) [reply source/fundamentals/atlas-search.txt:349]: A symptom of relying too much on Copilot autocomplete.

File: source/includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs [hunk 3]
```diff
@@ -130,6 +130,17 @@ public static List<Guitar> GeoWithinSearch()
         return result;
     }
 
+    public static List<Guitar> InSearch()
+    {
+        // start-in-search
+        var guitarList = new[] { "Fender", "Gibson" };
+        var result = guitarsCollection.Aggregate()
+            .Search(Builders<Guitar>.Search.In(g => g.Make, guitarList))
```
Comment (BorisDog) [inline source/includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs:138]: minor: could use new collection expression syntax:
`.Search(Builders<Guitar>.Search.In(g => g.Make, ["Fender", "Gibson"]))`

**Review by BorisDog — APPROVED** [review]
LGTM

**Pull request merged**